training/website_iterator.py

```diff
@@ -0,0 +1,30 @@
+from utils import text_extractor
+SIZE_OF_TRAINING_EXAMPLE = 5
+
+class WebsiteIterator:
+    def __init__(self, url):
+        self.url = url
+        self.content = self.get_website_text(url)
+        self.index = 0
+        self.content_length = len(self.content)
+
+    @staticmethod
+    def get_website_text(url):
+        all_text = ''
+        all_text = text_extractor.get_all_text_from_url(url)
+        return all_text
```

Comment:
what is the purpose of this method?
given the needed functionality required for the WebsiteIterator is supposed to be the private or public method?

Reply:
(hint - if it is private, which mean used only internally, so it should not be tested, it will be tested in-directly)

Reply:
(another hint, check how we mark private in python)